internal/elasticsearch/cluster/slm_test.go

```diff
@@ -68,7 +69,7 @@ resource "elasticstack_elasticsearch_snapshot_lifecycle" "test_slm" {
   snapshot_name = "<daily-snap-{now/d}>"
   repository    = elasticstack_elasticsearch_snapshot_repository.repo.name
 
-  indices              = ["data-*", "important"]
+  indices              = ["data-*", "abc"]
```

Comment:
String `set` is converted to list in alphabetical order, so replace it to make sure the result is not alphabetically ordered.
https://developer.hashicorp.com/terraform/language/expressions/type-constraints#conversion-of-complex-types